chad/README.md

```diff
@@ -0,0 +1,166 @@
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1` //true / js will change types when using ==
+
+**b)** `"1" === 1` //false / js will NOT change types when using ===
+
+**c)** `x == 'x'` //false / unless x has been set to 'x'
+
+**d)** `x == (x+'')` //true / == ignores the added string
+
+**e)** `'' == ' '` //false / an empty string is not equal to a string with a space in it
+
+**f)** `x = true` //true / sets x to true
+
+**g)** `var x; x == 'undefined'` //false / x == undefined, not the string 'undefined'
+
+**h)** `'9'<'10'` //false / alphabetically 1 is "less" than 9
+
+**i)** `typeof x + 1 === "number"` //false / typeof is unary so operates on the first thing it sees to it's right. in this case typeof x is "number", but then a 1 gets concatenated on the end so it's not === to "number"
+
+**j)** `typeof x % 2 === "number"` //false / in this case typeof x starts out as "number" but then the % 2 adds uncertainty so js says NaN???
+
+**k)** `typeof (x % 2) === "number"` //true / this time js operates in the () first and, assuming x was a number, will spit out a number to the typeof, resulting in  "number"
+
+**l)** `x++ == ++x` //false / pre-incrementing is happening on the right side of the expression, so they can never be equal. ex. x=3; x++(outputs 3 and sets x to 4) == ++x(sets x to 5 and outputs 5) will be false.
+
+**m)** `++x == x++` //true / pre-incrementing happens on the left only so... x=3; ++X(sets x to 4 and outputs 4) == x++(outputs 4 and then sets x to 5) both sides output 4 so the expression is true
+
+**n)** `"1"+x == 1+x` //false / "1"+x concatenates 1 and the value of x while 1+x adds 1 to the value of x
+
+**o)** `"0"+1 == 1` //true/ js will compare the string "01" with 1 and find it to be truthy
+
+**p)** `(typeof (x+1))===(typeof x)` //true / assuming x is a number, the left side will operate like **k)** above, and both sides will spit out "number"
+
+**q)** `(x*1 == x) || ((typeof x) != "number")` //true / the left side of the || returns a "boolean" which does not equal "number"
+
+**r)** `(x=(typeof (x+(typeof x))))==x`//true / the left side sets x = to "string" and then compares it to x on the right side which is now "string" thanks to the left side??
+
```

Comment:
More specifically, all the interior work here is set to x, and then is just compared against itself. So the right side isn't now anything, it's just what it always has been, which is x.